lib/modules/@apostrophecms/overlay/src/apos/components/TheApostropheOverlay.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="apos-overlay">
+    <div v-if="busy" class="busy"></div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'TheApostropheOverlay',
+  data() {
+    return {
+      busy: false
+    }
+  },
+  mounted() {
+    apos.bus.$on('apos-busy', state => (this.busy = state));
```

Comment:
I used `apos.bus.$on` and `apos.bus.$emit` in `lib/modules/@apostrophecms/utils/src/apos/always/http.js` because it felt less odd than doing:

`apos.utils.emit(document.body, 'apos-busy', false)` and here:

```js
document.body.addEventListener('apos-busy', state => (this.busy = state));
```

But both worked. So no problem to switch to the `addEventListener` approach if required.

(BTW, is `apos.bus` a Vue thing done by Apostrophe?)